client/app/dedicatedCloud/datacenter/drp/steps/ovh/dedicatedCloud-datacenter-drp-ovh.html

```diff
@@ -0,0 +1,45 @@
+<div data-ovh-alert="dedicatedCloudDatacenterDrpAlert"></div>
+
+<oui-message data-ng-if="$ctrl.drpInformations.state === 'toProvision' || $ctrl.drpInformations.state === 'provisionning' || $ctrl.isEnabling"
+             data-type="info">
+    <p data-translate="dedicatedCloud_datacenter_drp_confirm_creation_pending"></p>
+    <p data-translate="dedicatedCloud_datacenter_drp_confirm_creation_pending_task"></p>
+</oui-message>
+
+<div class="oui-progress-tracker my-5">
+    <ol class="oui-progress-tracker__steps">
+        <li class="oui-progress-tracker__step"
+            data-ng-class="{
+                'oui-progress-tracker__step_disabled': $ctrl.$stateParams.currentStep < 1,
```

Comment:
Will be nice if we add a 4th step, you could make it more generic :)

Reply:
It is static, sure, but it allows to keep it simple. If a new step have to be added, let's just add it as it is.

Reply:
And the day there are 200 steps ? :D

Reply:
This day, we'll build a freshly new site :D